### PR TITLE
Clean expired docs

### PR DESCRIPTION
**Pull request by SparkYuan**
1. delete expired docs
2. update cmd docs

### CODE REVIEW

File: docs/cmd/en/kusion_apply.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ Apply a configuration stack to resource(s) by work directory
 
 Apply a series of resource changes within the stack.
 
-Create or update or delete resources according to the KCL files within a stack. By default, Kusion will generate an execution plan and present it for your approval before taking any action.
+ Create or update or delete resources according to the KCL files within a stack. By default, Kusion will generate an execution plan and present it for your approval before taking any action.
```
Comment (howieyuen) [inline docs/cmd/en/kusion_apply.md:9]: redundant leading space? same below

File: docs/cmd/en/kusion_destroy.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ Destroy a configuration stack to resource(s) by work directory
 
 Delete resources by resource spec.
 
-Only KCL files are accepted. Only one type of arguments may be specified: filenames, resources and names, or resources and label selector.
+ Only KCL files are accepted. Only one type of arguments may be specified: filenames, resources and names, or resources and label selector.
```
Comment (howieyuen) [inline docs/cmd/en/kusion_destroy.md:9]: +1

File: docs/cmd/en/kusion_destroy.md [hunk 3]
```diff
@@ -6,9 +6,9 @@ Destroy a configuration stack to resource(s) by work directory
...
 
-Note that the destroy command does NOT do resource version checks, so if someone submits an update to a resource right when you submit a destroy, their update will be lost along with the rest of the resource.
+ Note that the destroy command does NOT do resource version checks, so if someone submits an update to a resource right when you submit a destroy, their update will be lost along with the rest of the resource.
```
Comment (howieyuen) [inline docs/cmd/en/kusion_destroy.md:11]: +1

File: docs/cmd/en/kusion_preview.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 
-Preview a series of resource changes within the stack.
+Preview a series of resource changes within the stack
 
 ### Synopsis
 
 Preview a series of resource changes within the stack.
 
-Create or update or delete resources according to the KCL files within a stack. By default, Kusion will generate an execution plan and present it for your approval before taking any action.
+ Create or update or delete resources according to the KCL files within a stack. By default, Kusion will generate an execution plan and present it for your approval before taking any action.
```
Comment (howieyuen) [inline docs/cmd/en/kusion_preview.md:9]: +1

**Review by howieyuen — APPROVED** [review]

**Review by markliby — APPROVED** [review]

**Pull request merged**